medicines/doc-index-updater/src/pars_upload.rs

```diff
@@ -2,21 +2,17 @@ use crate::{
     create_manager::models::BlobMetadata,
     document_manager::{accept_job, check_in_document_handler},
     models::{Document, FileSource},
+    multipart_form_data::*,
```

Comment:
How come * here instead of specifically named structs/functions?

Reply:
Just laziness. Have updated 😄